JesusFilm/core
# fix: add stop propagation

# Description


### Issue

links were being opened on radio options when selected in the editor. this should only happen when viewing the actual story.

[Link to Basecamp Todo](https://3.basecamp.com/3105655/buckets/37071143/todos/7380318983)

### Solution

readded stop propgation call that had accidentally been removed.

# External Changes

_If you have made changes to external services, need to add additional values to Doppler, or need to add something new to the database, explain it here. This may include updates to third-party services, changes to infrastructure configuration, integration with external APIs, etc._

# Additional information

to test: 
1.  go into the editor
2. create a radiobutton poll
3.  set a link for one one of the options
4. set a mail to address for another
5. click the options, they should not go to or open their respective links.
